src/DDTrace/Integrations/Curl/CurlIntegration.php

```diff
@@ -63,14 +63,48 @@ public static function load()
 
             $info = curl_getinfo($ch);
             $sanitizedUrl = Urls::sanitize($info['url']);
+            unset($info['url']);
+
             if ($globalConfig->isHttpClientSplitByDomain()) {
                 $span->setTag(Tag::SERVICE_NAME, Urls::hostnameForTag($sanitizedUrl));
             } else {
                 $span->setTag(Tag::SERVICE_NAME, 'curl');
             }
             $span->setTag(Tag::RESOURCE_NAME, $sanitizedUrl);
+
+
+            // Special case the Datadog Standard Attributes
+            //  https://docs.datadoghq.com/logs/processing/attributes_naming_convention/
+
             $span->setTag(Tag::HTTP_URL, $sanitizedUrl);
-            $span->setTag(Tag::HTTP_STATUS_CODE, $info['http_code']);
+
+            CurlIntegration::tagFromCurlInfo($span, $info, Tag::HTTP_STATUS_CODE, 'http_code');
+
+            // Datadog sets durations in nanoseconds - convert from seconds
+            $span->setTag('duration', $info['total_time'] * 1000000000);
+            unset($info['duration']);
+
+            CurlIntegration::tagFromCurlInfo($span, $info, 'network.client.ip', 'local_ip');
+            CurlIntegration::tagFromCurlInfo($span, $info, 'network.client.port', 'local_port');
+
+            CurlIntegration::tagFromCurlInfo($span, $info, 'network.destination.ip', 'primary_ip');
+            CurlIntegration::tagFromCurlInfo($span, $info, 'network.destination.port', 'primary_port');
+
+            CurlIntegration::tagFromCurlInfo($span, $info, 'network.bytes_read', 'size_download');
+            CurlIntegration::tagFromCurlInfo($span, $info, 'network.bytes_written', 'size_upload');
+
+
+            // Add the rest to a 'curl.' object
+            foreach ($info as $key => $val) {
+                // Datadog doesn't support arrays in tags
+                if (!is_array($val)) {
+                  // Datadog sets durations in nanoseconds - convert from seconds
```

Comment:
Style nit: comment needs one more space for indenting. I'm pretty sure that's the last style issue the linter picked up.